libevmasm/Instruction.h

```diff
@@ -97,6 +104,8 @@ enum class Instruction: uint8_t
 	MSIZE,				///< get the size of active memory
 	GAS,				///< get the amount of available gas
 	JUMPDEST,			///< set a potential jump destination
+	BEGINSUB,           ///< set a potential jumpsub destination -- not part of Instructions.cpp
+	BEGINDATA,          ///< begine the data section -- not part of Instructions.cpp
```

Comment:
Since these do not exist in the other list, they are not usable in inline assembly. Though it must be noted that if they are actually supported in `AsmCodeGen` (the binding to libevmasm), then they should be added to the list in order to the assembly text output to work properly.